gym-service/src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,16 @@
+spring:
+  config:
+    activate:
+      on-profile: "dev"
+  datasource:
+    url: jdbc:mariadb://localhost:5432/gym
+    username: root
+    password: root
+  jooq:
+    sql-dialect: org.mariadb.jdbc.Driver
+  jpa:
+    hibernate:
+      ddl-auto: update
+
+  activemq:
+    broker-url: "tcp://aws-ec2-url:7034"
```

Comment:
do you run it on ec2? I think it's overkill for local development, prefer to use stand-alone or even docker container with running mq (which I think is best way for local). There's also possibility to use embedded message queues, but can't say if all of them have such ability. Personally, I use ElasticMQ embedded to replace AWS SQS

Reply:
oh no. I just put some example for running it in different environment